integration_tests/e2eReferDisabled/refer.cy.ts

```diff
@@ -217,6 +219,43 @@ context('Refer', () => {
     notFoundPage.shouldContain404H2()
   })
 
+  it("Doesn't show the programme history page for a submitted referral", () => {
```

Comment:
Would this be clear if it stated explicitly that the user sees a "not found page" or "404 error"?

Reply:
I think it would - and is a good pattern to follow for future situations like this. For now, I'm going to leave it as-is as this "refer-disabled" suite will soon be removed once we turn off the feature flag, and to keep it consistent with all the other tests in this file, but definitely something to address in future.